client/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 12.10
+* Bump version of content-api-models to 12.9
```

Comment:
`(fixes namespacing of the new chart atom)`

Reply:
👍